mon-pix/app/components/routes/campaigns/assessment/skill-review.hbs

```diff
@@ -131,17 +131,16 @@
     <SkillReviewImprove @improve={{this.improve}}/>
   {{/if}}
 
-
   {{#unless @model.campaign.isForAbsoluteNovice}}
 
   <main class="skill-review-result-detail__content">
     <div class="skill-review-result-detail__table-header">
       <h2 class="skill-review-result-detail__subtitle">
         {{t 'pages.skill-review.details.title'}}
       </h2>
-      <CircleChart @value={{@model.campaignParticipationResult.masteryPercentage}}>
+      <CircleChart @value={{this.masteryPercentage}}>
```

Comment:
J'ai l'impression qu'on doit avoir `this.masteryRate` ici d'après le getter de `mon-pix/app/components/routes/campaigns/assessment/skill-review.js`

Reply:
Du coup on voit pas le "donut" se remplir : 
![image](https://user-images.githubusercontent.com/516360/132217591-ef64a625-ba3f-4c46-95e5-3fa447cc87d0.png)